src/enc/encode.rs

```diff
@@ -340,6 +341,17 @@ value: u32) -> i32 {
     params.favor_cpu_efficiency = value != 0;
     return 1i32;
   }
+  if p as (i32) == BrotliEncoderParameter::BROTLI_PARAM_BYTE_ALIGN as (i32) {
```

Comment:
I think `if p == BrotliEncoderParameter::BROTLI_PARAM_BYTE_ALIGN` should compile on rust 1.12?